en/views/cells.rst

```diff
@@ -276,3 +276,30 @@ This will allow us to define the option when creating the cell::
 Cell options are handy when you want data available as properties allowing you
 to override default values.
 
+Using Helpers inside a Cell
+===========================
+
+Cells have their own context and their own View instance but Helpers loaded inside your
+`AppView::initialize()` function are still loaded as usual.
+
+Loading a specific Helper just for a specific cell can be done via the following example::
+
+    namespace App\View\Cell;
+
+    use Cake\View\Cell;
+    use Cake\Datasource\Paginator;
+
+    class FavoritesCell extends Cell
+    {
+        protected $_validCellOptions = [];
+
+        public function initialize(): void {
+            $this->viewBuilder()->addHelper('MyCustomHelper');
+        }
+
+        public function display()
+        {
+            // Your code
+        }
```

Comment:
you can remove `display()`  function / `$_validCellOptions` and `use Cake\Datasource\Paginator;`  from sample

Reply:
I thought I would stay with the same example used a little bit above but this seems fine as well